src/plugins/dam/components/asset-snippet/Image.jsx

```diff
@@ -23,17 +23,21 @@ const Image = ({ asset, previewUrl }) => (
       </FormGroup>
     </div>
     <div className="col-12 col-md-6">
+      {console.log('Asset: ', JSON.parse(asset))}
+      {!previewUrl && <div className="spinner-wrapper"> <Spinner className="m-0 mr-2" color="#08a0e8" width="24" strokeWidth="8" /> </div>}
```

Comment:
@jalopez13 are custom styles on spinner really needed here?  those are global styles.  i would use spinner with virtually no customizations if possible, just use the builtin admin-ui or bootstrap classes

Reply:
Removed custom styling.